expat/configure.ac

```diff
@@ -395,9 +395,18 @@ AC_SUBST([AM_CFLAGS])
 AC_SUBST([AM_CXXFLAGS])
 AC_SUBST([AM_LDFLAGS])
 
+dnl Force the use of CMAKE_SHARED_LIBRARY_PREFIX under CMake
```

Comment:
With my picky hat on: I think we're "emulating" here rather than "forcing" it. We control the whole template.
Maybe that comment is even a leftover from the first try? My vote for either a to the point or no comment. Attention is a scarce resource.